src/components/ReduxFormCheckbox/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, ValidationFormGroup } from '@edx/paragon';
+
+const DEFAULT_CHECKBOX_LABEL = 'Notify learners';
+
+const ReduxFormCheckbox = ({
+  input,
+}) => (
+  <ValidationFormGroup
+    for="active"
+  >
+    <Form.Check
+      {...input}
+      type="checkbox"
+      id="active"
+      name="active"
+      className="ml-3"
+      checked={input.checked}
+      label={DEFAULT_CHECKBOX_LABEL}
```

Comment:
This checkbox should be reuseable; the label should be passed in from the Field component (as it is for the similar Textarea component, etc.)
The constant should live where the other constants for this form live, and be called something more specific, like `NOTIFY_LEARNERS_LABEL`

Reply:
(I believe if you just set the label on the Field component it will do the right thing and pass it to this checkbox.)

Reply:
FWIW the labels for these Fields are hard coded within the Bulk enrollment modal index. I'll move all of them to the constants file.